lib/format.js

```diff
@@ -26,10 +26,7 @@ module.exports = function format(style) {
         return sortObject(layer, sameOrderAs(reference.layer));
     });
 
-    var str = beautify(JSON.stringify(style), {
-        indent_size: 2,
-        keep_array_indentation: true
-    }).replace(/"filter": {[^}]+}/g, function (str) {
+    var str = stringify(style).replace(/"filter": {[^}]+}/g, function (str) {
```

Comment:
With `json-stringify-pretty-compact `, can we remove all of the custom string processing and just do `var str = stringify(style)`? What's the diff look like if you make that change and reformat mapbox-gl-styles?